docs/external/vfile-plugin.md

```diff
@@ -29,35 +29,52 @@ Detailed documentation can be found on our [GitHub Documentation Page](http://vm
 The recommended way to install vFile plugin is from docker cli:
```

Comment:
would like to make sure about ```All docker swarm managers should open [official etcd ports](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt) 2379 and 2380 (User-defined port number feature is WIP)``` <--- is this addressed through https://github.com/vmware/docker-volume-vsphere/pull/1988?

Reply:
That's right. Let me make another update.

Reply:
Should I wait for another PR or supply comments I have here? I show you have merged.

Reply:
I will submit another PR. Sorry I merged too fast...

Reply:
:)